qiskit/circuit/library/standard_gates/u3.py

```diff
@@ -26,7 +26,7 @@ class U3Gate(Gate):
     Implemented using two X90 pulses on IBM Quantum systems:
 
     .. math::
-        U2(\phi, \lambda) = RZ(\phi+\pi/2) RX(\frac{\pi}{2}) RZ(\lambda-\pi/2)
+        U3(\theta, \phi, \lambda) = RZ(\phi + \pi/2) RX(\theta) RZ(\lambda - \pi/2)
```

Comment:
This also has a mistake. The intention was to write equation 19 here https://arxiv.org/pdf/1612.00858.pdf. So the change should be:
```suggestion
        U3(\theta, \phi, \lambda) = RZ(\phi - \pi/2) RX(\pi/2) RZ(\pi - \theta) RX(\pi/2) RZ(\lambda - \pi/2)
```